src/vs/editor/standalone-languages/csharp.ts

```diff
@@ -51,7 +51,7 @@ export var language = <ILanguage> {
 		'internal', 'private', 'abstract', 'sealed', 'static', 'struct', 'readonly',
 		'volatile', 'virtual', 'override', 'params', 'get', 'set', 'add', 'remove',
 		'operator', 'true', 'false', 'implicit', 'explicit', 'interface', 'enum',
-		'null', 'async', 'await'
+		'null', 'async', 'await','fixed','sizeof','stackalloc','unsafe'
 		],
```

Comment:
This change is good, but note that this file is not in use in VS Code. For CSharp colorization in VS Code we use `/extensions/csharp/syntaxes/csharp.json`. 
This file is used for the standalone editor build.